### ISSUE TITLE
Order of conformance class is not as per the testng.xml in EARL report.

### ISSUE THREAD

**Issue by keshav-nangare** [opengeospatial/teamengine#171]
I have observed the gml32 earl result, the order of conformance classes is not similar to testng.xml in EARL report.

I have attached the gml32 earl report and the testng.xml .

[report.zip](https://github.com/opengeospatial/teamengine/files/352173/report.zip)

It would be better if we create report in sequence, so the HTML report is also having the same order.

**Comment by bermud** [comment]
Three options:
1. Use the english labels to order the conformance classes and tests ( A.1, A.2 etc..)
2. Generate in the rdf report an order list: e.g. http://www.snee.com/bobdc.blog/2014/04/rdf-lists-and-sparql.html
3. Hardcode the order of the conformance classes, and put the file somewhere in the project  folder. Maybe under scripts/report. A general xslt will import all the files found in the directories for all the tests under scripts. The xslt will then read the order of the conformance classes to represent them in the correct order.

Note: all the test have a dependency that can be calculated to figure out the order.

**Comment by bermud** [comment]
Richard suggested a week ago to use option 2. As discussed with Keshav it seems like a good idea.
@rjmartell, please implement the Seq list RDF solution.

```
<cite:TestRun>
 <cite:requirements>
   <rdf:Seq>
     <rdf:li rdf:resource="TestRequirement-1"/>
     <rdf:li rdf:resource="TestRequirement-2"/>
     <rdf:li rdf:resource="TestRequirement-3"/>
   </rdf:Seq>
 </cite:requirements>
</cite:TestRun>

```

**Comment by rjmartell** [comment]
Implemented in 4.9. The ordering of the conformance classes is the same as in the TestNG suite definition, but this does not necessarily reflect execution order. Sample output from a test run is shown below.

```
<cite:TestRun>
  <dct:extent rdf:datatype="http://www.w3.org/2001/XMLSchema#duration">PT4M40.188S</dct:extent>
  <cite:testsSkipped rdf:datatype="http://www.w3.org/2001/XMLSchema#int">24</cite:testsSkipped>
  <cite:testsFailed rdf:datatype="http://www.w3.org/2001/XMLSchema#int">8</cite:testsFailed>
  <cite:testsPassed rdf:datatype="http://www.w3.org/2001/XMLSchema#int">313</cite:testsPassed>
  <dct:identifier>b1154693-2733-49fa-a951-e43126a1b7e6</dct:identifier>
  <cite:requirements>
    <rdf:Seq>
      <rdf:li>
        <earl:TestRequirement rdf:about="Preconditions">
          <cite:testsSkipped rdf:datatype="http://www.w3.org/2001/XMLSchema#int">0</cite:testsSkipped>
          <cite:testsFailed rdf:datatype="http://www.w3.org/2001/XMLSchema#int">0</cite:testsFailed>
          <cite:testsPassed rdf:datatype="http://www.w3.org/2001/XMLSchema#int">3</cite:testsPassed>
          <dct:hasPart rdf:resource="org/opengis/cite/iso19142/SuitePreconditions#serviceIsAvailable"/>
          <dct:hasPart rdf:resource="org/opengis/cite/iso19142/SuitePreconditions#dataAreAvailable"/>
          <dct:hasPart rdf:resource="org/opengis/cite/iso19142/SuitePreconditions#verifyServiceDescription"/>
          <dct:title>Preconditions</dct:title>
        </earl:TestRequirement>
      </rdf:li>
      ...
```
